packages/fcl/src/current-user/exec-service/index.js

```diff
@@ -20,6 +21,11 @@ export async function execService({service, msg = {}, opts = {}, config = {}}) {
     ...config,
     services: await configLens(/^service\./),
     app: await configLens(/^app\.detail\./),
+    client: {
+      fclVersion: VERSION,
```

Comment:
Is `fclVersion` still being sent at the root level? I think you mentioned that you were going to keep it to be backwards compatible.

Reply:
Yup still being sent: https://github.com/onflow/fcl-js/blob/master/packages/fcl/src/current-user/exec-service/strategies/http-post.js#L10-L19

Reply:
Nice 👍

Reply:
Yeah left it in @psiemens . We could deprecate it at some point, but it's pretty minimal and could be useful in other contexts. Worth a discussion on what to do with it.